aiogram/api/types/inline_keyboard_markup.py

```diff
@@ -18,5 +18,49 @@ class InlineKeyboardMarkup(MutableTelegramObject):
     Source: https://core.telegram.org/bots/api#inlinekeyboardmarkup
     """
 
-    inline_keyboard: List[List[InlineKeyboardButton]]
+    row_width: int = 3
```

Comment:
Custom fields should be placed after all fields described in API docs with separating by one empty line in due to mixing the content make updating auto-generated code harder.